node/package.json

```diff
@@ -32,7 +32,7 @@
     "slugify": "^1.2.6",
     "typescript": "3.9.7",
     "unescape": "^1.0.1",
-    "vtex.search-graphql": "http://vtex.vtexassets.com/_v/public/typings/v1/vtex.search-graphql@0.44.0/public"
+    "vtex.search-graphql": "https://thalyta--carrefourbr.myvtex.com/_v/private/typings/linked/v1/vtex.search-graphql@0.45.0+build1625836827/public"
```

Comment:
Leaving this message here just to remember us to change it when the search-graphql version is released